components/shared/organisms/Header/Header.tsx

```diff
@@ -15,26 +15,36 @@ interface Props {
 }
 
 const Header = ({ user, signOut }: Props): JSX.Element => {
-  const links = [
-    { text: 'Profile', url: PROFILE, testId: 'profile' },
-    { text: 'Activity', url: ACTIVITY, testId: 'activity' },
-  ];
-
-  const actions = [
-    { text: 'Sign Out', onClick: signOut, testId: 'sign-out' },
-    { text: 'Log out from all devices', onClick: () => signOut({ everywhere: true }) },
-  ];
-
+  const [signIn] = useSignIn();
+  const signInCallable = signIn as CallableFunction;
```

Comment:
Could you please type this mutation (something like (this)[https://github.com/fs/react-next-base/pull/178/files#diff-e9524e7ead62c016572ce6956b3e37fe5d3dc0c023ee373ba12d8c44d6914fd4R1])